workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -51,6 +54,17 @@ public void Awake()
                 connectionConfig.UseExternalIp = workerConfigurations.UseExternalIp;
 #endif
             }
+#if UNITY_ANDROID
+            else if (Application.isMobilePlatform)
+            {
+                var worker = WorkerRegistry.CreateWorker<UnityClient>($"Client-{Guid.NewGuid()}", new Vector3(0, 0, 0));
+                Workers.Add(worker);
+                if (DeviceInfo.IsEmulator())
+                {
+                    connectionConfig = ReceptionistConfig.CreateConnectionConfigForAndroidEmulator();
```

Comment:
are we not setting any connection config in case the device is a phone?

Reply:
This will be covered in a separate PR which deals with local connection for physical devices. :)

Reply:
Shall we add a comment denoting that?